javascript/ql/src/semmle/javascript/security/dataflow/CleartextLogging.qll

```diff
@@ -13,44 +13,48 @@ module CleartextLogging {
   import CleartextLoggingCustomizations::CleartextLogging
 
   /**
-   * A dataflow tracking configuration for clear-text logging of sensitive information.
+   * A taint tracking configuration for clear-text logging of sensitive information.
    *
    * This configuration identifies flows from `Source`s, which are sources of
    * sensitive data, to `Sink`s, which is an abstract class representing all
    * the places sensitive data may be stored in clear-text. Additional sources or sinks can be
    * added either by extending the relevant class, or by subclassing this configuration itself,
    * and amending the sources and sinks.
    */
-  class Configuration extends DataFlow::Configuration {
+  class Configuration extends TaintTracking::Configuration {
```

Comment:
How does this work out for the slugs I provided earlier (https://git.semmle.com/gist/esben/6edfe4176c8b62241c7fa6784b54107a)? Does the `isSanitizerEdge` addition prevent the flood of FPs in practice (I would expect so)?

Reply:
I'll do a dist-compare soon, and then we'll find out.